hypothesis-python/tests/conjecture/test_utils.py

```diff
@@ -320,3 +320,7 @@ def test_can_draw_arbitrary_fractions(p, b):
 def test_samples_from_a_range_directly():
     s = cu.check_sample(range(10 ** 1000), "")
     assert isinstance(s, range)
+
+
+def test_p_continue_to_average_saturates():
+    assert cu._p_continue_to_avg(1.1, 100) == 100
```

Comment:
Can we replace this special case with an `@example` on `test_p_continue_to_average`?

Reply:
We do actually have such an example, but *also* need this test to reliably get full coverage from our conjecture-coverage task (the other is in nocover).